lib/routes.js

```diff
@@ -72,6 +72,10 @@ appRouter.get('/p/:shortid', response.showPublishSlide)
 appRouter.get('/p/:shortid/:action', response.publishSlideActions)
 // gey my note list
 appRouter.get('/api/notes/myNotes', noteController.listMyNotes)
+// delete note by id
+appRouter.delete('/api/notes/:noteId', noteController.deleteNote)
+// update note content by id
+appRouter.put('/api/notes/:noteId', urlencodedParser, noteController.updateNote)
```

Comment:
Use json parser instead

```suggestion
appRouter.put('/api/notes/:noteId', bodyparser.json(), noteController.updateNote)
```

Reply:
Fix.